src/miner.cpp

```diff
@@ -473,6 +474,11 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const MinerAddre
             } else {
                 monitoring_pool_balances = false;
             }
+            if (pindexPrev->nChainOrchardValue) {
+                orchardValue = *pindexPrev->nChainOrchardValue;
+            } else {
+                monitoring_pool_balances = false;
```

Comment:
This should always already be false, because Orchard balances are monitored whenever Sapling balances are, correct?

Reply:
That should indeed be the case for now, but it might not be in future (if e.g. Sapling is deprecated), and this is cheap code.